[2022-08-15 20:56:34,429] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG.Ingest_from_food_consumption_estimates manual__2022-08-15T20:56:28.970676+00:00 [queued]>
[2022-08-15 20:56:34,446] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG.Ingest_from_food_consumption_estimates manual__2022-08-15T20:56:28.970676+00:00 [queued]>
[2022-08-15 20:56:34,447] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-15 20:56:34,448] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-15 20:56:34,450] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-15 20:56:34,470] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): Ingest_from_food_consumption_estimates> on 2022-08-15 20:56:28.970676+00:00
[2022-08-15 20:56:34,485] {standard_task_runner.py:52} INFO - Started process 1546 to run task
[2022-08-15 20:56:34,538] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG', 'Ingest_from_food_consumption_estimates', 'manual__2022-08-15T20:56:28.970676+00:00', '--job-id', '1046', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpjo40jfms', '--error-file', '/tmp/tmp7z13pq8z']
[2022-08-15 20:56:34,550] {standard_task_runner.py:80} INFO - Job 1046: Subtask Ingest_from_food_consumption_estimates
[2022-08-15 20:56:34,743] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG.Ingest_from_food_consumption_estimates manual__2022-08-15T20:56:28.970676+00:00 [running]> on host 42abdd2e54ec
[2022-08-15 20:56:34,992] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG
AIRFLOW_CTX_TASK_ID=Ingest_from_food_consumption_estimates
AIRFLOW_CTX_EXECUTION_DATE=2022-08-15T20:56:28.970676+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-15T20:56:28.970676+00:00
[2022-08-15 20:56:43,424] {logging_mixin.py:115} INFO - Successfully uploaded food-consumption-estimates/food_table1.xlsx to s3-bucket-raw-usda
[2022-08-15 20:56:45,376] {logging_mixin.py:115} INFO - Successfully uploaded food-consumption-estimates/food_table2.xlsx to s3-bucket-raw-usda
[2022-08-15 20:56:46,421] {logging_mixin.py:115} INFO - Successfully uploaded food-consumption-estimates/food_table3.xlsx to s3-bucket-raw-usda
[2022-08-15 20:56:46,424] {python.py:173} INFO - Done. Returned value was: None
[2022-08-15 20:56:46,561] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG, task_id=Ingest_from_food_consumption_estimates, execution_date=20220815T205628, start_date=20220815T205634, end_date=20220815T205646
[2022-08-15 20:56:46,605] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-15 20:56:46,635] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
